frontend/src/Editor/Components/FilePicker.jsx

```diff
@@ -14,6 +14,7 @@ export const FilePicker = ({
   styles,
 }) => {
   //* properties definitions
+  const instructionText = component.definition.properties.instructionText.value ?? '';
```

Comment:
Hey @manishkushare , file picker components in existing apps don't have the instructionText property, so it is `undefined` for them, this is causing crashes for existing apps. Could you make this change and make sure that it is not crashing existing file picker widgets?

```suggestion
  const instructionText = component.definition.properties.instructionText?.value ?? '';
```